src/Http/Validation/NewRole.php

```diff
@@ -53,6 +53,7 @@ public function rules()
         return [
             'title' => 'string|unique:roles,title|required',
             'description' => 'string',
+            'permissions' => 'array',
```

Comment:
Sounds like to be a little bit too permissive. Should add an explicit constraint based on available permissions